src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTests.cs

```diff
@@ -697,7 +828,7 @@ private void TestSimpleDecrypt_RoundTrip(CertLoader certLoader, ContentInfo cont
             VerifySimpleDecrypt(encodedMessage, certLoader, expectedContentInfo);
         }
 
-        private void VerifySimpleDecrypt(byte[] encodedMessage, CertLoader certLoader, ContentInfo expectedContent)
+        internal void VerifySimpleDecrypt(byte[] encodedMessage, CertLoader certLoader, ContentInfo expectedContent)
```

Comment:
Protected, perhaps?

Reply:
@bartonjs that was my first try but the `CertLoader` parameter type is internal. I suppose it could be `private protected` but when I checked there was exactly zero uses of that so I stuck with `internal`.